src/main/resources/biz_de.properties

```diff
@@ -237,7 +237,7 @@ DeleteTenantJobFactory.deletingMainEntity = Lösche das Datenobjekt des Mandante
 DeleteTenantJobFactory.description = Löscht einen ausgewählten Mandanten und alle zugehörigen Daten.
 DeleteTenantJobFactory.label = Mandant löschen
 DeleteTenantJobFactory.simulateInfo = Die Löschung wird nur simuliert.
-DeleteTenantJobFactory.takeAction = Wirklich löschen?
+DeleteTenantJobFactory.takeAction = Löschen Simulieren
```

Comment:
Also rename the actual key to `DeleteTenantJobFactory.simulate` and `DeleteTenantJobFactory.simulate.help`?